dateutil/parser.py

```diff
@@ -1067,6 +1065,18 @@ def _parse(self, timestr, dayfirst=None, yearfirst=None, fuzzy=False,
 DEFAULTPARSER = parser()
 
 
+def skip_token(skipped_tokens, last_skipped_token_i, i, l):
```

Comment:
This should not be a public module function like this. Can you move it to a private method on `parser`?

I also don't love how it both mutates `skipped_tokens` and returns a value, but that is a secondary concern and if it's private we won't be married to that particular design decision.